src/web_crawler/twir_crawler.rs

```diff
@@ -155,7 +165,7 @@ impl TwirCrawler {
         };
 
         for element in found {
-            info!("Found: {} -> {}", element.title, element.link.0);
+            info!("Found[{}]: {} -> {}", search_mode, element.title, element.link.0);
```

Comment:
This would flood the logging because it is done in a nested loop. Could you please remove it? I don't think this will be helpful for debugging later. Can be kept as a trace if you want to but it will make things harder to find.